src/azul/terra.py

```diff
@@ -646,19 +646,26 @@ def for_registered_user(cls, authentication: OAuth2) -> 'TDRClient':
     def drs_client(self) -> DRSClient:
         return DRSClient(http_client=self._http_client)
 
-    def get_duos(self, source: TDRSourceRef) -> Optional[MutableJSON]:
+    def get_duos(self,
+                 source: TDRSourceRef
+                 ) -> tuple[str | None, MutableJSON | None]:
```

Comment:
When one is None so is the other. You should be able to express that in the return type hint.